README.md

```diff
@@ -1074,6 +1074,7 @@ _Related: [Groupware](#groupware)_
 
 _Related: [Note-taking & Editors](#note-taking--editors), [Wikis](#wikis)_
 
+- [Atomic Server](https://github.com/atomicdata-dev/atomic-data-rust) - Knowledge graph database with documents (similar to Notion), tables, search, and a powerful linked data API. Lightweight, very fast and no runtime dependencies. ([Demo](https://atomicdata.dev/)) `MIT` `Rust`
```

Comment:
Please change the language to `Docker/Rust`

Reply:
Done!